tekton-pipelines/base/tasks/kaniko-task.yaml

```diff
@@ -65,6 +65,7 @@ spec:
       args:
         - --dockerfile=$(resources.inputs.app.path)/$(params.docker_file)
         - --context=$(resources.inputs.app.path)/$(params.docker_context)
+        - --build-arg=ENVIRONMENT=$(ENVIRONMENT)
```

Comment:
is it coming from config map?
if so I'd like to change it to use params of the task, https://github.com/saritasa-nest/saritasa-devops-tekton/blob/41189fb8d33e28ce630cee396c939dfb4773b57d/tekton-pipelines/base/tasks/kaniko-task.yaml#L26

that way we have a consistent naming of
dev, staging, prod as env names. 

otherwise the configmap could have values with different names, like production, develop, development etc.

Reply:
Yes, but we have `dev` environment in build params, but I need to use `development` cause I have a `development.txt` requirements file there), that's why I implemented it in this way. So I will have to use `if` condition in Dockerfile to make it work

Reply:
I understand but why no to ask Roman to name it this (our way)?